docs/pages/menus/index.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+
+  <DocsPageTemplate>
+
+    <DocsPageSection title="Overview" anchor="#overview">
+
+      <p>Menus allow users to make a selection from a list of options. We use two types of menus:</p>
+
+      <h3>
+        Dropdown menu
+      </h3>
+      <p>
+        Allow users to discover and access complex actions. They can be generated from buttons, icons, and in some cases by right-clicking a location
```

Comment:
Here and a few other places, there are `.` periods missing at the end of paragraph sentences.